networkparams_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2016 The btcsuite developers
+// Copyright (c) 2016 The Decred developers
```

Comment:
It's 2018 and this is new code for Decred, so there shouldn't be any copyright attributed to btcsuite.